gradle_modify/modify_gradle.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+ARTIFACT_ID="idflakies"
+ARTIFACT_VERSION="1.0.3-SNAPSHOT"
```

Comment:
Please use the versions matching what I changed in [this commit](https://github.com/idflakies/iDFlakies/commit/4bb34655975f687ea80c50af6e74cab760a21ad0)